TestsFoundation/TestsFoundation/Extensions/XCUIApplicationExtensions.swift

```diff
@@ -17,34 +17,36 @@
 import Foundation
 import XCTest
 
+// warning: These functions will actually use `app` for finding elements,
+// so will only function properly on the test target app.
 public extension XCUIApplication {
     func find(label: String, type: XCUIElement.ElementType = .any) -> Element {
        return XCUIElementWrapper(
-            descendants(matching: type)
+            app.descendants(matching: type)
```

Comment:
this is the trade-off for the recomputing `XCUIElement`, we need to explicitly reference an `XCUIApplication` here. `self` wouldn't be recomputed, and `XCUIApplication` doesn't have a `bundleIdentifier` to make sure we're creating a new one for the same app. For now we are assuming you are always working with the test app.